osidb/sync_manager.py

```diff
@@ -542,6 +542,77 @@ class BZSyncManager(SyncManager):
     Sync manager class for OSIDB => Bugzilla synchronization.
     """
 
+    @classmethod
+    def schedule(cls, sync_id, *args, **kwargs):
+        """
+        Schedule BZSyncManager's sync_task to Celery queue.
+
+        TODO:
+          - This is a workaround. SyncManager.schedule should not
```

Comment:
If this is only a workaround, we should create a new Jira to track the correct change so we do not forget about it.

Reply:
Filed as OSIDB-3205 and it will probably become a part of a new epic based on our future team talks about parallelism-related design changes.